tests/CMakeLists.txt

```diff
@@ -59,7 +53,7 @@ if(ENABLE_TESTS)
             "SYSREPO_REPOSITORY_PATH=${PROJECT_BINARY_DIR}/test_repositories/${test_name}"
             "SYSREPO_SHM_PREFIX=_tests_sr_${test_name}"
         )
-        if(${CMAKE_VERSION} VERSION_GREATER "3.7")
+        if(${CMAKE_VERSION} VERSION_GREATER_EQUAL "3.7")
```

Comment:
I know `cmake` version 3.7 supports this already but `cmake` version < 3.7 does not support `VERSION_GREATER_EQUAL` hence me using `VERSION_GREATER`. I do not really consider it a problem.

Reply:
Ah, good catch, thanks, I'll fix this one.